src/site/xdoc/server/config-tika.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+<document>
+
+    <properties>
+        <title>Apache James Server 3 - Tika Configuration</title>
+    </properties>
+
+    <body>
+
+        <section name="Tika Configuration">
+
+            <p>When using ElasticSearch, you can configure an external Tika server for extracting and indexing text from attachments.
+                Thus you can significantly improve user experience upon text searches.</p>
+
+            <p>Note that to use this feature you need Guice, built with ElasticSearch</p>
+
+            <p>Consult <a href="https://github.com/apache/james-project/blob/master/dockerfiles/run/guice/cassandra/destination/conf/tika.properties">tika.properties</a> to get some examples and hints.</p>
+
+            Here are the different properties:
+
+            <dl>
+                <dt><strong>tika.host</strong></dt>
+                <dd>IP or domain name of your Tika server. The default value is 127.0.0.1</dd>
+                <dt><strong>tika.port</strong></dt>
+                <dd>Port of your tika server. The default value is 9998</dd>
+                <dt><strong>tika.timeoutInMillis</strong></dt>
+                <dd>Timeout when issuing request to the tika server. The default value is 3 seconds.</dd>
+            </dl>
+
+            Note: You can launch a tika server using this command line:
+
+            <code><pre>docker run --name tika logicalspark/docker-tikaserver:1.15rc2</pre></code>
```

Comment:
maybe adding a warning about slowness (and desactivating some indexes) would be a good thing